ompi/mca/common/monitoring/common_monitoring.h

```diff
@@ -94,6 +94,13 @@ static inline int mca_common_monitoring_get_world_rank(int dst, struct ompi_comm
     return ret;
 }
 
+/* Translate the rank from the given communicator of a process to its rank in MPI_COMM_RANK. */
+static inline int mca_common_monitoring_get_world_rank(int dst, struct ompi_communicator_t*comm,
+                                                       int*world_rank)
```

Comment:
Why rename this function, and leave a 1-line inline alias function to get to the new function?  Why not just call the new function name directly?